core/services/ocrbootstrap/delegate.go

```diff
@@ -118,14 +118,15 @@ func (d *Delegate) ServicesForSpec(jobSpec job.Job, qopts ...pg.QOpt) (services
 		if routerFields.ContractVersion != 1 || routerFields.ContractUpdateCheckFrequencySec == 0 {
 			return nil, errors.New("invalid router contract config")
 		}
-		configProvider, err = relayer.NewFunctionsProvider(
+		configProvider, err = relayer.NewPluginProvider(
 			ctx,
 			types.RelayArgs{
 				ExternalJobID: jobSpec.ExternalJobID,
 				JobID:         spec.ID,
 				ContractID:    spec.ContractID,
 				RelayConfig:   spec.RelayConfig.Bytes(),
 				New:           d.isNewlyCreatedJob,
+				ProviderType:  string(job.OCR2Functions),
```

Comment:
@jmank88 @krehermann Was this the kind of change you had in mind? (Here taken explicitly, but taken from the job spec in the case of Median and Mercury)

I've coerced the value to a string since that way it's acts as a passthrough value when we source it from the providerType.

Reply:
Oh I didn't realize this one was such an oddball :thinking:
Yeah, I think this works for now, and we should be able to evolve in a compatible way :thinking: